### PR TITLE
Loan Purpose Options Call documentation

### PR DESCRIPTION
**Pull request by nnekkanti**

### CODE REVIEW

File: funding/disbursements.rst [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+        "9": "Buy, lease or repair equipment, technology or vehicles",
+        "10": "Buy or lease storage",
+        "11": "Cover routine business costs",
+        "12": "Expand or remodel my physical business space",
+        "13": "Hire employees",
+        "14": "Open another location",
+        "15": "Pay for marketing or advertising expenses",
+        "16": "Purchase inventory or supplies",
+        "17": "Purchase another business",
+        "18": "Update my web or mobile presence"
```
Comment (erichhansen) [inline funding/disbursements.rst]: This does not match other `OPTIONS` operations, such as from v2/profile/persons, v2/profile/persons/{personId}}/verification, or v2/profile/business. Should look soomething like `{ "POST": { ... "values": { 1: "example" ...`

File: funding/disbursements.rst [hunk 2]
```diff
@@ -1,5 +1,43 @@
 Disbursements
 =============
+.. _loal-purpose-options:
```
Comment (erichhansen) [inline funding/disbursements.rst]: This is missing updates to initiate disbursement and get loans.

**Comment by erichhansen** [comment]
@nnekkanti Make sure this gets updated with the latest contract.

**Comment by nnekkanti** [comment]
@erichhansen this is updated as per the latest contract which will have LoanPurposeId

**Comment by erichhansen** [comment]
@nnekkanti Can you work on getting this merged in? I would like to see @alan-rodriguez's comments above addressed.

**Comment by nagendranekkanti** [comment]
@alan-rodriguez I have update the DOC for options call. Any changes for the details?

File: funding/disbursements.rst [hunk 3]
```diff
@@ -1,5 +1,111 @@
...
+**Request**
+
+::
+
+    OPTIONS /v2/Funding/{user-id}/Disbursement
+    Authorization: Bearer Q0xJRU5UX0lEOkNMSUVOVF9TRUNSRVQ=
+
+**Response**
+
+InitiateDisbursement POST contarct whith list of LoanPurpose.
```
Comment (alan-rodriguez) [inline funding/disbursements.rst]: Spelling: 'contract' instead of 'contarct'

Comment (alan-rodriguez) [reply funding/disbursements.rst]: @nnekkanti fix this and then we can close this.

**Pull request merged**